### PR TITLE
feat: add support for ActionImport and FunctionImport

### PR DESCRIPTION
**Pull request by drktfl**
Introduces a separate type representing both `<ActionImport>` and `<FunctionImport>` (similar to the single type representing both `<Action>` and `<Function>`).

I hope that this can improve robustness when looking up the actions from annotations - `UI.DataFieldForAction`  (and hopefully be useful in the mockserver, too).

Another option (hack!) could be to erase the ActionImports by adding the referenced Action under the ActionImports FQN into the set of actions. The drawback of this option is that the converted data would end up with actions that actually do not exist (because they do not have their original FQN any longer).

### CODE REVIEW

**Comment by nlunets** [comment]
The code looks nice but i'm unsure about the concept behind and the implication there are with it, can you quickly explain that here ?

**Comment by drktfl** [comment]
In the existing implementation the annotation converter fails to resolve unbound actions when processing DataFieldForAction annotations. That is, the `DataFieldForAction.ActionTarget` property stayed undefined.

TL;DR: the goal primarily was to fix that.

But there is another aspect: Unbound _actions_ can only be called if they are exposed as _action imports_ in the entity container. This implies that DataFieldForAction annotations must use the _action import_ path in the annotation, not the _action_.

With the exiting implementation, both got mixed up in terms of Action vs. ActionImport, but also OData 2 vs. OData 4 (both have a FunctionImport, but they work differently). So it is not that straightforward to separate them in the converter. The converter might resolve something that is not callable from an OData perspective.

I think implications on other components are limited and mostly affect consumers of the annotation converter. They must provide the additional `actionImports` property when using it. If they just provide an empty array to satisfy type constraints, it will fail to resolve unbound actions, as before. Other than that, the action target resolution now works much better 😃 

Also, the action parameters have an `isEntitySet` flag. It does not have any effect and probably can be removed? #188

**Review by nlunets — APPROVED** [review]

**Review by nlunets — APPROVED** [review]
